app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,75 @@
+class TweetsController < ApplicationController
+
+
+    get '/tweets/new' do
+      if logged_in?
+        erb :'/tweets/create_tweet'
+      else
+        redirect to "/login"
+      end
+    end
+
+    get '/tweets' do
+      if !logged_in?
+        redirect to "/login"
+      else
+        @user = current_user
+        @tweets = Tweet.all
+        erb :'/tweets/tweets'
+      end
+    end
+
+    post '/tweets' do
+      if logged_in? && !params[:content].empty?
+        @tweet = Tweet.create(content: params[:content], user_id: session[:user_id])
+        redirect to "/tweets/#{@tweet.id}"
+      else
+        redirect to "/tweets/new"
+      end
+    end
+
+    get '/tweets/:id' do
+      if logged_in?
+        @user = current_user
+        @tweet = Tweet.find_by(user_id: @user.id)
+        erb :'/tweets/show_tweet'
+      else
+        redirect to "/login"
+      end
+    end
+
+    get '/tweets/:id/edit' do
+      if logged_in?
+        @user = current_user
+        @tweet = Tweet.find_by(user_id: @user.id)
```

Comment:
Nice job taking the tweet's user into account! However, it looks like this will actually find the first tweet that belongs to @user rather than finding the tweet specified in the url.